main_startup.rb

```diff
@@ -0,0 +1,63 @@
+class App
+  def initialize
+    @books = []
+    @music_albums = []
+    @games = []
+    @genres = [Genre.new('Blues'), Genre.new('Classical Music'), Genre.new('Hip hop'), Genre.new('Rap'),
```

Comment:
You instantiate new Genre-based objects, however, there is no class called Genre.